tasks/src/Zero-shot-classification/data.ts

```diff
@@ -0,0 +1,30 @@
+import type { TaskDataCustom } from "../Types";
```

Comment:
Your data.ts template is missing two sections: demo & metrics. Can you add them back (can be empty) because the file has a template that needs to be followed.